lighty-modules/lighty-netconf-sb/src/main/java/io/lighty/modules/southbound/netconf/impl/util/NetconfUtils.java

```diff
@@ -73,7 +73,7 @@ private NetconfUtils() {
     public static InstanceIdentifier<Node> createNetConfNodeMountPointII(final NodeId nodeId) {
         KeyedInstanceIdentifier<Topology, TopologyKey> instanceIdentifier =
                 InstanceIdentifier.create(NetworkTopology.class).child(Topology.class,
-                        new TopologyKey(new TopologyId(TopologyNetconf.QNAME.getLocalName())));
+                        new TopologyKey(new TopologyId(NetconfNodeUtils.DEFAULT_TOPOLOGY_NAME)));
```

Comment:
Revert this back, `NetconfNodeUtils.DEFAULT_TOPOLOGY_NAME` is marked deprecated and will be removed.